charts/sftp/README.md

```diff
@@ -0,0 +1,100 @@
+# Sftp Helm Chart
+
+Helm Chart for setting up a sftp server
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Sftp Helm Chart](#sftp-helm-chart)
+  - [Prerequisites](#prerequisites)
+  - [Chart Details](#chart-details)
+    - [Verification](#verification)
+    - [Installation](#installation)
+    - [Example](#example)
+  - [Assigning a external IP to the SFTP](#assigning-a-external-ip-to-the-sftp)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Prerequisites
+
+* Kubernetes 1.8+ with beta APIs enabled
+
+## Chart Details
+
+This chart will do the following:
+
+* deploying an instance of a sftp-server on a K8s cluster.
+
+### Verification
+
+Lint the chart
+
+```
+$ helm lint
+```
+
+To test the generated k8s resources from the helm chart
+
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
+```
+
+### Installation
+
+To install the chart with the release name `my-service` using a dedicated namespace(recommended):
+
+```
+$ helm upgrade --install --namespace my-shop-namespace my-service -f values.yaml .
+```
+
+The chart can be customized using the following configurable parameters:
+
+|          Parameter                 |                Description                 |                   Default                   |
```

Comment:
Is there any required parameters, if yes I think we should mark those as required ?

Reply:
87ec092